sdk/communication/Azure.Communication.Rooms/README.md

```diff
@@ -1,23 +1,21 @@
 # Azure Communication Rooms client library for .NET
 
 This package contains a C# SDK for the Rooms Service of Azure Communication Services.
+Azure Communication Services (ACS) Rooms is a set of APIs, used by Contoso server applications to create a server-managed conversation space with fixed set of lifetime and participants, pre-defining rules from server-tier both who and when can communicate (like scheduled meeting creation).
 
-Rooms overview
-    Azure Communication Services (ACS) Rooms is a set of APIs, used by Contoso server applications to create a server-managed conversation space with fixed set of lifetime and participants, pre-defining rules from server-tier both who and when can communicate (like scheduled meeting creation).
+With the General Availability preview release of ACS Rooms, Contoso will be able to:
```

Comment:
What do you mean by the GA preview release?

Reply:
This is our GA candidate, meaning that it will first be in preview to collect feedback and then we aim to release this as GA version. I realize that `General Availability preview` is a confusing name, I'll update to `With the preview release of ACS Rooms...`, but let me know if there's a better name for this